docs/standard/serverless-architecture/azure-serverless-platform/azure-functions.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Azure Functions | Serverless apps. Architecture, patterns, and Azure implementation.
+description: Azure functions provide serverless capabilities across multiple languages (C#, JavaScript, Java, and more) and platforms to provide event-driver instant scale code.
+author: JEREMYLIKNESS
+ms.author: jeliknes
+ms.date: 3/21/2018
+ms.prod: .net
+ms.technology: dotnet
+ms.topic: article
+---
+# Azure Functions
+
+Azure functions provide a serverless compute experience. A function is invoked by a *trigger* (such as access to an HTTP endpoint or a timer) and executes a block of code or business logic. Functions also support specialized *bindings* that connect to resources like storage and queues. There are two versions of the Azure Functions framework. The legacy version supports the full .NET Framework and the new runtime supports cross-platform .NET Core applications. Additional languages besides C# such as JavaScript, F#, and Java are supported. Functions created in the portal provide a rich scripting syntax. Functions created as standalone projects can be deployed with full platform support and capabilities.
+
+For more information, see: [Azure Functions documentation](/azure/azure-functions).
+
+## Functions v1 vs. v2
+
+There are two versions of the Azure Functions runtime: 1.x and 2.x. Version 1.x is generally available. It supports .NET development from the portal or Windows machines and uses the .NET Framework. 1.x supports C#, JavaScript, and F#, with experimental support for Python, PHP, TypeScript, Batch, Bash, and PowerShell.
+
+Version 2.x is in preview. It leverages .NET Core and supports cross-platform development on Windows, macOS, and Linux machines. 2.x does not support F#, adds support for Java, and does not yet support any of the experimental languages. Version 2.x uses a new binding extensibility model that enables third-party extensions to the platform, independent versioning of bindings, and a more streamlined execution environment.
+
+> **There is a known issue in 1.x with [binding redirect support](https://github.com/Azure/azure-functions-host/issues/992).** The issue is specific to .NET development. Projects with dependencies on libraries that are a different version from the libraries included in the runtime are impacted. The functions team has committed to make concrete progress on the problem. The team will address binding redirects in 2.x before it goes into general availability.
+
+For more information, see: [Compare 1.x and 2.x](/azure/azure-functions/functions-versions).
+
+## Programming language support
+
+The following languages are supported either in general availability, preview, or experimental.
+
+|Language      |1.x         |2.x      |
+|--------------|------------|---------|
+|**C#**        |GA          |Preview  |
+|**JavaScript**|GA          |Preview  |
+|**F#**        |GA          |No       |
```

Comment:
> No

This should probably be "Preview", since it's supported but you do need to hand-write your own function.json

Reply:
This table is taken directly from here: https://docs.microsoft.com/en-us/azure/azure-functions/supported-languages I'll remove "no" since there is support from workaround and leave it blank like the original table.